Resources/doc/index.md

```diff
@@ -30,6 +30,17 @@ sumo_coders_framework_multi_user:
     prefix:   /
 ```
 
+Add the logout route:
+
+```yaml
+#app/config/security.yml
+firewalls:
```

Comment:
mss hier al beginnen bij `security:`, dat wordt in de andere voorbeelden ook gedaan.